### PR TITLE
BREAKING CHANGE: Clean up files in repo

### PR DESCRIPTION
**Pull request by UziTech**
**Marked version:** 8.0.0

## Description

Clean up files that are not needed in repo. I left `marked.min.js` since we need it in the repo to run our demo on master and specific commits and prs.

This is only breaking if someone uses the files in lib/ on the master branch. Technically they were never meant to be used so I'm not sure we actually need to bump the major version.

## Contributor

- [ ] Test(s) exist to ensure functionality and minimize regression (if no tests added, list tests covering this PR); or,
- [x] no tests required for this PR.
- [ ] If submitting new feature, it has been documented in the appropriate places.

## Committer

In most cases, this should be a different person than the contributor.

- [ ] CI is green (no forced merge required).
- [ ] Squash and Merge PR following [conventional commit guidelines](https://www.conventionalcommits.org/).

### CODE REVIEW

**Review by calculuschild — APPROVED** [review]
This looks ok to me. Keeping in mind I'm not as familiar with all of the packaging steps, but this change makes sense to me.

File: .gitignore [hunk 1]
```diff
@@ -3,5 +3,7 @@
 node_modules/
 test/compiled_tests
 public
+lib
 docs/LICENSE.md
 vuln.js
+man/marked.1
```
Comment (styfle) [inline .gitignore:9]: Lets add `marked.min.js` too

File: docs/build.js [hunk 2]
```diff
@@ -24,6 +25,8 @@ async function init() {
   console.log('Cleaning up output directory ' + outputDir);
   await rm(outputDir, { force: true, recursive: true });
   await mkdir(outputDir);
+  console.log(`Copying file ${join(cwd, 'LICENSE.md')}`);
+  console.log(`          to ${join(inputDir, 'LICENSE.md')}`);
   await copyFile(join(cwd, 'LICENSE.md'), join(inputDir, 'LICENSE.md'));
```
Comment (styfle) [inline docs/build.js:30]: Let's add something like:

```
await copyFile(join(cwd, 'marked.min.js'), join(outputDir, 'marked.min.js''));
```

Then in the client side JS when you select latest, you can `fetch(new URL('/marked.min.js', window.location))`.

Comment (UziTech) [reply docs/build.js:30]: Is there a way to get the PR vercel link with just a PR#?

Comment (UziTech) [reply docs/build.js:29]: I found it based on the vercel comment View Preview link

Comment (UziTech) [reply docs/build.js:29]: [PR Demo](https://marked-website-git-fork-uzitech-remove-built-markedjs.vercel.app/demo/?text=%23%20test&options=%7B%0A%20%22async%22%3A%20false%2C%0A%20%22breaks%22%3A%20false%2C%0A%20%22extensions%22%3A%20null%2C%0A%20%22gfm%22%3A%20true%2C%0A%20%22hooks%22%3A%20null%2C%0A%20%22pedantic%22%3A%20false%2C%0A%20%22silent%22%3A%20false%2C%0A%20%22tokenizer%22%3A%20null%2C%0A%20%22walkTokens%22%3A%20null%0A%7D&version=pr%3A2963)

Comment (UziTech) [reply docs/build.js:29]: or we don't even need the pr version because now we can go to the preview and use /marked.min.js

Comment (styfle) [reply docs/build.js:29]: Exactly, we don’t need an input anymore. Either you select an old version or you select “HEAD”. And HEAD for a production deployment is master branch and for a preview deployment it’s the PR branch.

**Review by styfle — APPROVED** [review]
Great work, thanks! 🎉

**Pull request merged**